### PR TITLE
5 Reasons to Sponsor the Perl Toolchain Summit

### PR DESCRIPTION
**Pull request by oalders**
- **Tidy CONTRIBUTING.md**
- **Add dependeny installation to CONTRIBUTING.md**
- **Update olaf-alders author data**
- **Add an article about sponsoring the Perl Toolchain Summit**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,32 +1,48 @@
...
 ## Write an article
 
-Interested in writing an article for Perl.com? Perhaps you want to get the word out about your new startup, provide a tutorial on your favorite module, or have community news to share. This document is for you.
+Interested in writing an article for Perl.com? Perhaps you want to get the word
+out about your new startup, provide a tutorial on your favorite module, or have
+community news to share. This document is for you.
 
-You can propose an article as a [GitHub issue](https://github.com/tpf/perldotcom/issues), or simply send up a pull request. Inside the repo, you can run the `make new` target and you'll be led through the questions to set up your initial article template:
+You can propose an article as a [GitHub
+issue](https://github.com/tpf/perldotcom/issues), or simply send up a pull
```
Comment (briandfoy) [inline CONTRIBUTING.md:12]: update that link

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,32 +1,48 @@
...
+whether that's a simple typo to adjusting code for new interfaces.
 
-You can edit files directly in GitHub, which will automatically fork our repo and send a pull request.
+You can edit files directly in GitHub, which will automatically fork our repo
+and send a pull request.
 
-For big edits, you might want to raise a [GitHub issue](https://github.com/tpf/perldotcom/issues) before you spend too much time on it.
+For big edits, you might want to raise a [GitHub
+issue](https://github.com/tpf/perldotcom/issues) before you spend too much time
+on it.
```
Comment (briandfoy) [inline CONTRIBUTING.md:46]: update this link

**Review by briandfoy — CHANGES_REQUESTED** [review]
There are a couple of links to fix up.

**Comment by oalders** [comment]
@briandfoy, thanks! I updated and force-pushed.

**Comment by oalders** [comment]
And I think that stuff is better off in its own PR, so I moved it to #362

**Pull request merged**